### PR TITLE
[SEB-2614] dockerize app and deploy to ECS

### PR DESCRIPTION
**Pull request by shuynh**
https://jira.nypl.org/browse/SEB-1847

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,68 +39,44 @@ script:
...
-  secret_access_key: "$AWS_SECRET_ACCESS_KEY_PRODUCTION"
-  region: us-east-1
-  app: my-library-nyc-app
-  env: my-library-nyc-app-react-production
-  bucket_name: elasticbeanstalk-us-east-1-946183545209
-  bucket_path: my-library-nyc-app-react-production
-  on:
-    repo: NYPL/MyLibraryNYCApp
-    branch: production
+- stage: deploy qa
```
Comment (gkallenberg) [inline .travis.yml:42]: Does the development branch deployment still need space here? A development ECS stack could be created on the `nypl-sandbox` account which is the development account for LSP, and the `nypl-digital-dev` account acts as the qa and production account.

Comment (gonuguntla) [reply .travis.yml:42]: @gkallenberg DIscussed with Stephen today we don't need Development server for MylibraryNyc. Here no need to give space.

File: .travis.yml [hunk 2]
```diff
@@ -38,68 +39,44 @@ script:
...
+    - LOCAL_TAG_NAME=mylibrarynycapp:$TRAVIS_BRANCH-$TRAVIS_BUILD_NUMBER
+    - REMOTE_FULL_URL=$REMOTE_IMAGE_URL:$TRAVIS_BRANCH-latest
+    - CLUSTER_NAME=mylibrarynycapp-$TRAVIS_BRANCH
+    install:
+    - pip install --user awscli
+    - docker login --username simplye --password $DOCKER_LOGIN_PASSWORD
+    script:
+    - if [ ${TRAVIS_BRANCH} = "production" ]; then export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID_PRODUCTION;
+      export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY_PRODUCTION; else export
+      AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID_QA; export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY_QA;
```
Comment (gonuguntla) [inline .travis.yml:73]: @gkallenberg @shuynh any idea  when docker is going to  deploy on QA?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+WORKDIR $APP_HOME
+
+## bundle
+RUN gem install bundler -v '2.2.28' 
+RUN bash -lc "rvm use ruby-$RUBY_VERSION --default"
+#RUN bash -lc "rvm install ruby-$RUBY_VERSION && rvm use ruby-$RUBY_VERSION --default"
+RUN bundle config --global github.https true \
+    && bundle install
+
+FROM production AS development
```
Comment (gonuguntla) [inline Dockerfile]: we don't need development server

**Pull request closed without merging**